manager/integration/tests/test_recurring_job.py

```diff
@@ -483,3 +483,39 @@ def test_recurring_jobs_for_detached_volume():  # NOQA
     17. cleanup
     """
     pass
+
+
+@pytest.mark.skip(reason="TODO")
+def test_recurring_jobs_on_nodes_with_taints():  # NOQA
+    """
+    Test recurring jobs on nodes with taints
+
+    Context:
+
+    longhorn/longhorn#1903
+
+    Steps:
+
+    1. Set taint toleration for Longhorn components
+       `persistence=true:NoExecute`
+    1. Taint `node-1` with `persistence=true:NoExecute`
+    1. Create a volume, vol-1.
+       Attach vol-1 to node-1
+       Write some data to vol-1
+    1. Create a recurring backup job which runs every minute
```

Comment:
What's the retain count of the recurring job here, more than 1?

Reply:
Thank you! Fixed